authnid/make_db.py

```diff
@@ -0,0 +1,29 @@
+import psycopg2
+import psycopg2.extras
+
+def connect_db(uri):
+    conn = psycopg2.connect(uri, cursor_factory=psycopg2.extras.DictCursor)
+    conn.set_session(autocommit=True)
+    return conn
+
+def make_cursor(connection):
+    return connection.cursor()
+
+def make_db(config):
+    connection = connect_db(config['DATABASE_URI'])
+    cursor = make_cursor(connection)
+    cursor.execute('''
+        CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+        CREATE TABLE IF NOT EXISTS users (
+            id uuid DEFAULT uuid_generate_v4() PRIMARY KEY,
+            email VARCHAR,
```

Comment:
unique?

Reply:
Would DOD ID also be unique? And are we certain there are no legit cases where we'd have redundant emails? Like if someone were issued a new DOD ID but their email doesn't change?